<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDlg</class>
 <widget class="QDialog" name="ConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QPushButton" name="cmdSave">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>121</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdClose">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>121</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QPushButton" name="cmdAppDataPath">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>browse ..</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>App Data Path:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAppDataPath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cboStartup">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>56</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>56</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>At Startup:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtAppDataPath</tabstop>
  <tabstop>cmdAppDataPath</tabstop>
  <tabstop>cboStartup</tabstop>
  <tabstop>cmdSave</tabstop>
  <tabstop>cmdClose</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
